src/pudl/analysis/plant_parts_eia.py

```diff
@@ -365,6 +366,38 @@
 ]
 
 
+@asset(
+    name="mega_generators_eia",
+    compute_kind="Python",
+)
+def mega_gens_asset(
+    mcoe_generators_yearly: pd.DataFrame, denorm_ownership_eia860: pd.DataFrame
+) -> pd.DataFrame:
+    """Create mega generators table asset."""
+    return MakeMegaGenTbl().execute(
+        mcoe=mcoe_generators_yearly,
+        own_eia860=denorm_ownership_eia860,
+    )
+
+
+@asset(
```

Comment:
It would be nice to be able to generate the PPE for only a subset of years, how have we been doing that type of thing? Add `context` with a start and end year? Ideally you'd be able to run the fast ETL and only get those 2 years of PPE.

Reply:
What happens if you try and generate the PPE in the fast ETL right now? It'll only have the last couple of years of data available in the database to work with won't it?

Reply:
I guess I mean if you have full data in the database (with all the years) but you're making changes to the PPE and want to generate a new PPE with a couple years of data but don't want to wait the full 45 min

Reply:
Ah, okay. Hmm. A 45 minute asset that runs at the end of the DAG is going to be a bottleneck too.  I wonder if there's anything we can do to split it up or speed it up to avoid extending the whole ETL by 45 minutes and also make it quick and easy to regenerate the whole thing on the fly in development.  I'm not sure how to twiddle input parameters like this on a per-asset per-run basis but I assume it's possible.

Reply:
It probably wouldn't be hard to parallelize. Currently it loops through each plant part and aggregates records to that part level, then concatenates into one dataframe. This loop could be parallelized. But probably worth making that a separate issue/PR at this point.

Reply:
Agree, sounds like a good follow-up issue. Too many 45-minute assets will definitely get frustrating!